mango-cli/src/main/scala/org/bdgenomics/mango/cli/VizReads.scala

```diff
@@ -419,49 +362,27 @@ class VizReads(protected val args: VizReadsArgs) extends BDGSparkCommand[VizRead
     def initReference() = {
       val referencePath = Option(args.referencePath)
 
-      VizReads.referencePath =
+      VizReads.referencePath = {
         referencePath match {
           case Some(_) => referencePath.get
           case None    => throw new FileNotFoundException("reference file not provided")
         }
-
-      VizReads.refRDD = new ReferenceRDD(sc, VizReads.referencePath)
+      }
+      VizReads.refRDD = new ReferenceMaterialization(sc, VizReads.referencePath, 1000)
+      VizReads.chunkSize = VizReads.refRDD.chunkSize
```

Comment:
chunkSize says 1000 not 5000. I don't think this should be a command line arg but maybe it should change based on resources.